<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>462</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Beam Destination</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" rowspan="3">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Expert Screens</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="PyDMShellCommand" name="PyDMShellCommand">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>MR1L0</string>
        </property>
        <property name="commands" stdset="0">
         <stringlist>
          <string>pydm --hide-menu-bar --hide-nav-bar -m '{&quot;BASE_PV&quot;:&quot;MR1L0:HOMS&quot;}' $launcher_dir/homs_overview/mirrorScreen.py</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PyDMShellCommand" name="PyDMShellCommand_2">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>MR2L0</string>
        </property>
        <property name="commands" stdset="0">
         <stringlist>
          <string>pydm --hide-menu-bar --hide-nav-bar -m '{&quot;BASE_PV&quot;:&quot;MR2L0:HOMS&quot;}' $launcher_dir/homs_overview/mirrorScreen.py</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PyDMShellCommand" name="PyDMShellCommand_3">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>MR1L3</string>
        </property>
        <property name="commands" stdset="0">
         <stringlist>
          <string>pydm --hide-menu-bar --hide-nav-bar -m '{&quot;BASE_PV&quot;:&quot;MR1L3:HOMS&quot;,&quot;OUT_STATE&quot;: &quot;TRUE&quot;}' $launcher_dir/homs_overview/mirrorScreen.py</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PyDMShellCommand" name="PyDMShellCommand_4">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>MR2L3</string>
        </property>
        <property name="commands" stdset="0">
         <stringlist>
          <string>pydm --hide-menu-bar --hide-nav-bar -m '{&quot;BASE_PV&quot;:&quot;MR2L3:HOMS&quot;}' $launcher_dir/homs_overview/mirrorScreen.py</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item>
       <widget class="PyDMShellCommand" name="PyDMShellCommand_5">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>MR1L4</string>
        </property>
        <property name="commands" stdset="0">
         <stringlist>
          <string>pydm --hide-menu-bar --hide-nav-bar -m '{&quot;BASE_PV&quot;:&quot;MR1L4:HOMS&quot;,&quot;OUT_STATE&quot;:&quot;TRUE&quot;}' $launcher_dir/homs_overview/mirrorScreen.py</string>
         </stringlist>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>1000</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Mirror Motion</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QComboBox" name="energyComboBox">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="hutchComboBox">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="moveButton">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Go!</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Desired Beam Destination</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Desired Energy Range</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="maximumSize">
      <size>
       <width>800</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Mirror Readback</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Current Beam Destination</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Current Energy Range</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>145</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Current Photon Energy</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="beamLabel">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>28</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="energyLabel">
        <property name="minimumSize">
         <size>
          <width>220</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="PyDMLabel" name="currEnergyLabel">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>140</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="textFormat">
         <enum>Qt::PlainText</enum>
        </property>
        <property name="showUnits" stdset="0">
         <bool>true</bool>
        </property>
        <property name="precisionFromPV" stdset="0">
         <bool>false</bool>
        </property>
        <property name="channel" stdset="0">
         <string>ca://PMPS:LFE:PE:UND:CurrentPhotonEnergy_RBV</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Motion Indicators</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>MR1L0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>MR2L0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>MR1L3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>MR2L3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>MR1L4</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="StatusIndicator" name="MR1L0_status" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="StatusIndicator" name="MR2L0_status" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="StatusIndicator" name="MR1L3_status" native="true"/>
      </item>
      <item row="1" column="3">
       <widget class="StatusIndicator" name="MR2L3_status" native="true"/>
      </item>
      <item row="1" column="4">
       <widget class="StatusIndicator" name="MR1L4_status" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMLabel</class>
   <extends>QLabel</extends>
   <header>pydm.widgets.label</header>
  </customwidget>
  <customwidget>
   <class>PyDMShellCommand</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.shell_command</header>
  </customwidget>
  <customwidget>
   <class>StatusIndicator</class>
   <extends>QWidget</extends>
   <header>Helper_widgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
